src/SignalR/server/Core/src/Internal/Proxies.cs

```diff
@@ -147,6 +147,7 @@ internal sealed class SingleClientProxy<THub> : ISingleClientProxy where THub :
     public SingleClientProxy(HubLifetimeManager<THub> lifetimeManager, string connectionId)
     {
         _lifetimeManager = lifetimeManager;
+        ArgumentNullException.ThrowIfNull(connectionId, nameof(connectionId));
```

Comment:
Since the param name would be injected by `CallerArgumentExpressionAttribute`, I'd like to suggest removing it.

```suggestion
        ArgumentNullException.ThrowIfNull(connectionId);
```

Reply:
Also, why not update UserProxy, GroupProxy, and GroupExceptProxy